inspec/a2-iam-v2-integration/controls/introspection.rb

```diff
@@ -56,20 +56,20 @@
     end
   end
 
-  describe 'introspect some with parameters' do
+  describe 'introspect (single) with parameters' do
```

Comment:
This is IntrospectParameterized, not IntrospectSome.

Reply:
maybe just "introspect with single parameter"? or just omit the "(single)" (it doesn't feel meaningful or clear here without additional context